Update webhook docs (#5994)
PostHog/posthog.com: contents/docs/webhooks/discord.md

showTitle: true
---

> You can receive messages in Discord whenever any of your specified [actions](/docs/user-guides/actions) are triggered. 

Reviewer: ```suggestion
You can send messages to Discord whenever an [actions](/docs/user-guides/actions) triggers. 
```

> You can receive messages in Discord whenever any of your specified [actions](/docs/user-guides/actions) are triggered. 

Author: ```suggestion
You can send messages to Discord whenever an [action](/docs/user-guides/actions) triggers. 
```

## How to setup the Discord webhook

### 1. Create an incoming webhook in Discord 

1. Navigate to the channel where you want to add the webhook and select the 'Edit Channel' option from left navigation pane. 
1. Select the 'Integrations' option from the left navigation pane. 
1. If you are creating a webhook for the first time, click the "Create Webhook" button. If you have other webhooks, click "View Webhooks" and then "New Webhook". 
2. Give the webhook any name you prefer. 

> 3. Select the channel where the message should be posted from the drop-down.

Reviewer: ```suggestion
3. Select the channel where the message should be posted from the dropdown.
```

4. Click the "Copy Webhook URL" button to copy the webhook URL into the clipboard.  

It should look something like this. 

![Discord Webhook setup](../../images/discord-webhook.png)

For more information, see the [Discord webhook docs](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks).

### 2. Setup the webhook in PostHog

> Copy the Webhook URL from Discord, navigate to project settings in PostHog paste the url into the "Webhook integration section".

Reviewer: ```suggestion
Copy the Webhook URL from Discord, navigate to project settings in PostHog, and paste the URL into the "Webhook integration section".
```
